packages/gatsby-theme-carbon/src/components/CardGroup/CardGroup.module.scss

```
@@ -0,0 +1,49 @@
+.row [class*="--col-lg"] {
+  border-top: 1px solid $carbon--gray-20;
+}
+
+.row [class*="--col-lg"]:nth-child(2n) {
+  @include carbon--breakpoint("lg") {
+    margin-right: 33%;
+  }
+}
+
+// Borders for cards at the medium breakpoint
+
+.row [class*="--col-lg"]:nth-child(-n + 2) {
+  @include carbon--breakpoint("md") {
+    border-top: 0px solid transparent;
+  }
+}
+
+.row [class*="--col-lg"]:nth-child(even) {
+  @include carbon--breakpoint("md") {
+    border-left: 1px solid $carbon--gray-20;
+  }
+}
+
+// Borders for dark cards at medium breakpoint
+
+.row
+  [class*="--col-lg"]:nth-child(-n + 2)
+  .#{$prefix}--resource-card.#{$prefix}--resource-card--dark {
+  border-top-color: $carbon--black-100;
+}
+
+// Borders for stacking MiniCards within a column
+
+.row [class*="--col-lg"] a:nth-child(n + 2) [class*="MiniCard-module--card--"] {
```

Ideally we don't wanna use the `[class*=` technique for rules we define in the theme. That's a good escape hatch for theme sites but in the theme itself, we want to avoid depending on class names generated in different modules.

One option would be having replacing this with a generic `.groupable-card` class and importing it into the various cards that can be used with the card group. This also allows someone to use `CardGroup` with a card they've defined rather than the one's we've hard-coded rules for.

In that same vein, we generally don't want to use elements in our styles if we can avoid it. There's no guarantee a card will be an anchor tag. We can use a wildcard there to say we're down with buttons, links, images, whatever.

```suggestion
.row [class*="--col-lg"] *:nth-child(n + 2) .groupable-card {
```

---

consider flattening rules, ideal: `.row .groupable-card:nth-child(n+2)`